storage/filesystem/object.go

```diff
@@ -36,9 +37,66 @@ func (s *ObjectStorage) Set(core.Object) (core.Hash, error) {
 	return core.ZeroHash, fmt.Errorf("not implemented yet")
 }
 
+func (s *ObjectStorage) Get(h core.Hash) (core.Object, error) {
+	var obj core.Object
+	var err error
+
+	obj, err = s.getFromObject(h)
```

Comment:
the function `getFromObject` needs a better name, it returns an object from an object?